bigtable/gc.go

```diff
@@ -35,7 +35,7 @@ const (
 // A GCPolicy represents a rule that determines which cells are eligible for garbage collection.
 type GCPolicy interface {
 	String() string
-	proto() *bttdpb.GcRule
+	Proto() *bttdpb.GcRule
```

Comment:
In general we try not to expose protos directly out from the manual layers. Also adding a new method to an interface is a breaking change in Go

Reply:
Agreed with @codyoss . IIRC, we discussed not exposing protos directly in #5993 . Why this breaking change?

Reply:
Pretend a user implemented your that interface for their type:

```go
type Foo struct {}
```